evil-collection-helm.el

```diff
@@ -165,10 +165,25 @@
     "%" 'helm-ff-run-query-replace-regexp
     "D" 'helm-ff-run-delete-file)       ; Ivy has "D".
 
+  ;; These helm bindings should always exist, the evil equivalents do
+  ;; nothing useful in the minibuffer (error or pure failure).
+  ;; RET can't do a second line in the minibuffer.
+  ;; The C-n/C-p completions error with 'helm in helm' session.
+  ;; C-o switches to evil state (again, not useful).
+  (evil-collection-define-key '(insert normal) 'helm-map
+    (kbd "RET") 'helm-maybe-exit-minibuffer
```

Comment:
I don't understand why we would need to bind this.  The bindings are already good for me and some Helms like Helm Find Files use a different binding (e.g. `helm-ff-RET`).
Can give an example?

Reply:
With this package as is, if I press RET in the minibuffer (with minibuffer setup enabled) during a helm session, it only works if I'm in INSERT mode, not in NORMAL mode, which is counter intuitive, as j / k scroll through results, it is a pain to then press i RET instead of just RET.

Reply:
If you go in minibuffer, type some text, press ESC to leave insert mode and go to NORMAL mode, pressing RET will select your entry?  I'll try testing in a vanilla/minimal but it may be a bit until I get to.

Reply:
Sorry for the delay.  You were right, I can reproduce now.